src/components/Card.js

```diff
@@ -48,14 +65,61 @@ export default class Card {
       });
   }
 
-  _handleLikeIcon() {
-    this._element
-      .querySelector(".cards__like-button")
-      .classList.toggle("cards__like-button_active");
-  }
-
-  _handleDeleteIcon() {
+  handleRemoveCard() {
     this._element.remove();
     this._element = null;
   }
+
+  _handleLikeIcon() {
+    if (
+      this._element
+        .querySelector(".cards__like-button")
+        .classList.contains("cards__like-button_active")
+    ) {
+      this.api
+        .dislikeCard(this._id)
+        .then((data) => {
+          this._element
+            .querySelector(".cards__like-button")
+            .classList.remove("cards__like-button_active");
+          localStorage.removeItem(`card-${this._id}-liked`);
```

Comment:
`localStorage` should not be used for **likes** because you need to check if it's liked using the `liked` param from the card object. 

If you visit the site from another browser, you'll not see your liked items becuase of `localStorage`